### ISSUE TITLE
Typescript breaks install using require() from standard nodejs

### ISSUE THREAD

**Issue by vaultec81** [dappkit/aviondb#35]
**Version**:
0.2.2

**Platform**:
Nodejs

**Describe the bug**
```
import Store from "./Store";
^^^^^^

SyntaxError: Cannot use import statement outside a module
...snip, traceback from parent code using require()
```

**To Reproduce**
1. npm install aviondb@0.2.2
2. Import aviondb using require()

**Comment by deanpress** [comment]
@vaultec81 What version of Node are you using?

**Comment by deanpress** [comment]
I'm confused because 0.2.2 was released _before_ the typescript migration. This new Typescript refactor should be a higher version.

**Comment by vasa-develop** [comment]
@deanpress Don't get confused by `0.2.2`. @vaultec81 must be using the `master` branch.

**Comment by deanpress** [comment]
You're using `require()` to import an uncompiled typescript dependency. If you're manually importing from github (and the github repo doesn't contain the latest compiled js package), you need to build it first in `node_modules` and then import the compiled js file instead. 

`package.json` isn't configured yet to point to the build file (`aviondb.min.js`).

Example for preparing typescript module for npm: https://codeburst.io/https-chidume-nnamdi-com-npm-module-in-typescript-12b3b22f0724

I made a PR that should make the package ready for npm: https://github.com/dappkit/aviondb/pull/36

**Comment by vasa-develop** [comment]
@vaultec81 I have merged @deanpress's PR. Feel free to check out the `master` module now & close the issue if everything goes ok :)

**Comment by vaultec81** [comment]
I was using 0.2.2 I think it must have been left off cached from when master was installed. Will run a test and check it out.

**Comment by vasa-develop** [comment]
@vaultec81 let us know if you are still facing this issue :wink:

**Comment by vaultec81** [comment]
A new issue after the update through git.
```
Error: Cannot find module 'node_modules/aviondb/dist/src/index.js'. Please verify that the package.json has a valid "main" entry
```

**Comment by vasa-develop** [comment]
@vaultec81  Yeah, that's because we don't push the `dist` folder to the Github repo. 
I have released a pre-release: [v0.2.3-rc.0](https://www.npmjs.com/package/aviondb?activeTab=versions)

I checked it out and it works.
```
// Install
npm i aviondb@next

// Usage
const AvionDB = require("aviondb").default

// OR

import AvionDB from "avoindb"
```

I'm working on some ways so that we don't need to add `.default` when using `require()`
